devel/mos/Portfile

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+categories          devel
+license             Apache-2.0
+name                mos
+
+set python.versions \
+                    38
```

Comment:
You've added a dependency on this version of python but you haven't informed the build system that it should use it. As a result, the port fails to build:
```
env: python3: No such file or directory
```